runtime/doc/syntax.txt

```diff
@@ -4678,7 +4678,7 @@ font={font-name}					*highlight-font*
 	When setting the font for the "Normal" group, this becomes the default
 	font (until the 'guifont' option is changed; the last one set is
 	used).
-	The following only works with Motif and Athena, not with other GUIs:
+	The following only works with Motif not with other GUIs:
```

Comment:
Ok. Which text below that is relevant to delete though? The `:` leaves me to believe its the rest of the text?

```text
The following only works with Motif not with other GUIs:
When setting the font for the "Menu" group, the menus will be changed.
When setting the font for the "Tooltip" group, the tooltips will be
changed.
All fonts used, except for Menu and Tooltip, should be of the same
character size as the default font!  Otherwise redrawing problems will
occur.
```